src/transformers/models/wav2vec2/configuration_wav2vec2.py

```diff
@@ -66,8 +68,8 @@ class Wav2Vec2Config(PretrainedConfig):
             The norm to be applied to 1D convolutional layers in feature extractor. One of :obj:`"group"` for group
             normalization of only the first 1D convolutional layer or :obj:`"layer"` for layer normalization of all 1D
             convolutional layers.
-        feat_extract_dropout (:obj:`float`, `optional`, defaults to 0.0):
-            The dropout probabilitiy for all 1D convolutional layers in feature extractor.
+        feat_proj_dropout (:obj:`float`, `optional`, defaults to 0.0):
```

Comment:
this was a wrong default - the `feat_proj_dropout` is pretty much never used in fairseq